packages/bpk-component-price-range/src/BpkPriceRange.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useEffect, useRef, useState } from 'react';
+
+import clamp from 'lodash/clamp';
+
+import BpkText, { TEXT_STYLES } from '../../bpk-component-text/src/BpkText';
+import { cssModules } from '../../bpk-react-utils';
+
+import BpkPriceMarker from './BpkPriceMarker';
+import { MARKER_TYPES } from './common-types';
+
+import type { MarkerType } from './common-types';
+
+import STYLES from './BpkPriceRange.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+type Marker = {
+  price: string;
+  percentage: number;
+};
+
+export type Props = {
+  min?: number;
+  max?: number;
+  showPriceIndicator?: boolean;
+  marker: Marker;
+  segments: {
+    low: Marker;
+    high: Marker;
+  };
+};
+
+const BpkPriceRange = ({
+  marker,
+  max = 100,
+  min = 0,
+  segments,
+  showPriceIndicator = false,
```

Comment:
according to the design [figma](https://www.figma.com/design/yN0hFyZlKL0Jwbpi0rEKYT/Backpack-Beta?node-id=27496-100&t=H9iEHgriisTp7B1C-0) , seems the `showPriceIndicator` default value should be true

Reply:
Yeah, I have mixed them up, thanks